doc/source/getting_started/code/Exploring Vector Spaces.py

```diff
@@ -0,0 +1,328 @@
+
```

Comment:
Generated. I can add a command to the generating script to strip the first blank line.